### DISCUSSION TITLE
Is the API documentation for 1.0.0-dev.2 out of date?

### DISCUSSION THREAD

**Discussion by wujek-srujek** [rrousselGit/riverpod#581] [Q&A] [answered]
I upgraded to 1.0.0-dev.2 and am using `select` without hooks, and it all works. However, the docs https://pub.dev/documentation/riverpod/1.0.0-dev.2/riverpod/ProviderBase/select.html still say that it isn't supposed to work in my case (no hooks, not `ProviderContainer.lister`. Are they out of date?

**Comment by rrousselGit** [comment, marked as answer]
Yes, I have yet to change the documentation
